clinics/repo.go

```diff
@@ -340,6 +340,34 @@ func (c *repository) DeletePatientTag(ctx context.Context, id, tagId string) (*C
 	return c.Get(ctx, id)
 }
 
+func (c *repository) ListMembershipRestrictions(ctx context.Context, id string) ([]MembershipRestrictions, error) {
+	clinic, err := c.Get(ctx, id)
+	if err != nil {
+		return nil, err
+	}
+
+	return clinic.MembershipRestrictions, nil
+}
+
+func (c *repository) UpdateMembershipRestrictions(ctx context.Context, id string, restrictions []MembershipRestrictions) error {
+	clinicId, _ := primitive.ObjectIDFromHex(id)
```

Comment:
I'm guessing id is already checked before being passed in here so any error can be ignored?

Reply:
ids are checked at against the api spec which expects them to be valid ObjectIDs